src/main/io/debugserial.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "platform.h"
+
+#define DBG_DEFAULT_BAUDRATE 115200
+#define DBG_DEFAULT_PORT SERIAL_PORT_USART3
+
+#ifdef USE_DEBUG_SERIAL
+	#define __DBG_SETRATE(x)    dbg_setRate(x)
+	#define __DBG_TICK()		dbg_tick()
+	#define __DBG_ENABLE()      dbg_enable()
+	#define __DBG_DISABLE()		dbg_disable()
+	#define __DBG_CLS()			dbg_clear()
+	#define	__DBG_ENTER()		dbg_enterFunction(__func__)
+	#define	__DBG_LEAVE()		dbg_leaveFunction(__func__)
+	#define __DBG_TEXT(x)		dbg_write(x)
+    #define __DBG_LINE(x)		dbg_writeLine(x)
+	#define __DBG_VALUEF(x, y)  dbg_valuef(x, y)
+	#define __printf(x) 		tfp_printf x
+	#define __DBG_EVERY(x,y) do {	\
```

Comment:
This is very specific to your debugging style. 
Also should be possible to rewrite it with `for`, so that `__DBG_EVERY(x) y`; is possible.